### ISSUE TITLE
GTX 1080 and RTX 2060 crash

### ISSUE THREAD

**Issue by kr-deps** [Lolliedieb/lolMiner-releases#51]
Clean Windows 10, newest NVIDIA driver.
The miner crashes after `Start Mining...`. Each card in isolation crashes, too.

```
Setup Miner...
Size:
 Device 0:
    Name:    GeForce RTX 2060
    Address: 1:0
    Vendor:  NVIDIA Corporation
    Memory:  6144 MByte
    Active:  true (Selected Algorithm: BeamHash III 3G / 4G)
 Device 1:
    Name:    GeForce GTX 1080
    Address: 7:0
    Vendor:  NVIDIA Corporation
    Memory:  8192 MByte
    Active:  true (Selected Algorithm: BeamHash III 6G)
 Device 2:
    Name:    Intel(R) HD Graphics 610
    Address: 4294967295:4294967295
    Vendor:  Intel(R) Corporation
    Memory:  3232 MByte
    Active:  false (not selected)

Connecting to pool...
Connected to beam.sunpool.top:3334 (TLS enabled)
TLS Handshake success
Authorized worker: 32f2e8765c2e8f5ea41becc5f397024c94d80cc5fc50ee917af23b260ecb3a5f.testRun
New job received: 95044 Difficulty 1024
Failed to obtain all required NVML function pointers
NVIDIA hardware monitoring disabled
Start Mining...
```

**Comment by kr-deps** [comment]
On another Windows installation (same hardware) the NVIDIA GPUs work though. But they do not always work with AMD devices selected.

Maybe this has to do with AMD+NVIDIA rig and both drivers installed?

**Comment by kr-deps** [comment]
There seems to be no pattern in when the miner crashes :(

Sometimes even AMD-only rig crashes and sometimes it doesn't. The miner needs testing on "diverse" rigs with different GPU models.

**Comment by Lolliedieb** [comment]
There is one crash I know that appears when the api is active - can you check configs if this may be the case? This silent crashes are hard to debug :/

**Comment by Lolliedieb** [comment]
Can you run on 1.03 an test if the problem is still there?

**Comment by thebrumby** [comment]
Hi, I have the same issue. Windows 10 64-bit with 2080 TI. I tried using DDU to remove all drivers and reinstalled the latest NVidia ones, which didn't make a difference. No on screen error and nothing recorded in the logs folder. Same installs work fine on my other rigs with same OS and AMD graphics cards. Thanks in advance.

**Comment by thebrumby** [comment]
![image](https://user-images.githubusercontent.com/29182343/87096267-5b11b400-c23a-11ea-80a6-7365435746b2.png)

**Comment by thebrumby** [comment]
Sometimes it manages to make one share, other times to just does the 10 second timer and close the window with no visible error.
![image](https://user-images.githubusercontent.com/29182343/87096437-be9be180-c23a-11ea-8b6c-c1a61b45ecfe.png)

**Comment by Lolliedieb** [comment]
So well temperature monitoring seems to fail, but else it should mine?
Or is the miner stuck?

**Comment by thebrumby** [comment]
Unfortunately within 10 seconds of "Start mining" it closes the window with no error. Sometimes it manages one share (is appears to be working fine for a start), but then no error, no log and just closes the window.

**Comment by thebrumby** [comment]
With my AMD card rigs (4 GPU + 6 GPU) it mines for weeks non-stop with a much better hash rate than other mining software. However on the machine with the RTX2080TI but same windows OS it crashes every time.

**Comment by Lolliedieb** [comment]
Should be fixed in 1.05 :)
